src/seedu/addressbook/commands/FindCommand.java

```diff
@@ -48,13 +49,34 @@ public CommandResult execute() {
      */
     private List<ReadOnlyPerson> getPersonsWithNameContainingAnyKeyword(Set<String> keywords) {
         final List<ReadOnlyPerson> matchedPersons = new ArrayList<>();
+        keywords = changeKeywordsintoLowerCase(keywords);
         for (ReadOnlyPerson person : addressBook.getAllPersons()) {
-            final Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
+            Set<String> wordsInName = new HashSet<>(person.getName().getWordsInName());
+            wordsInName = changeKeywordsintoLowerCase(wordsInName);
             if (!Collections.disjoint(wordsInName, keywords)) {
                 matchedPersons.add(person);
             }
         }
         return matchedPersons;
     }
 
+    /**
+     * Return keyword in lower case so as to make it case insensitive
+     * @param keyToChange contains keyword to be converted to lower case
+     * @return keywords into lower case
+     */
+
+    private static Set<String> changeKeywordsintoLowerCase(Set<String> keyToChange){
+        Set<String> removeStringSet = new HashSet<>();
+        Set<String> addStringSet = new HashSet<>();
+
+        for(String string : keyToChange){
+            removeStringSet.add(string);
+            addStringSet.add(string.toLowerCase());
+        }
```

Comment:
Scope for SLAP here. In the current logic, too much is happening in this method. Try to use slap to make the code self-readable.